.eslintrc.js

```diff
@@ -193,7 +220,6 @@ module.exports = {
         ],
         "import/no-named-default": "error",
         "import/no-relative-packages": "error",
-        "import/extensions": ["error", "never"],
```

Comment:
Since we're pulling this out, are we able to remove `"eslint-plugin-import": "^2.26.0",` from `package.json` and `packages/perseus/package.json`?

Reply:
Oh duh, never mind. Literally the line above this is using it. 🤦